src/pages/who-we-are/_OurOffices.tsx

```diff
@@ -126,7 +126,7 @@ const OurOffices = () => {
 
             <NumberSection columns="1fr 1fr 1fr 1fr" column_gap="120px" row_gap="4rem">
                 {our_offices_count.map(({ count, title }) => (
-                    <StyledFlex fd="column" key={title.props.translate_text}>
+                    <StyledFlex fd="column" key={title.props.translate_text || title}>
```

Comment:
the `title` in the `our_offices_count`, is a `ReactElement`, using it as `key` prop is not suitable.

Reply:
@michio-rework don't have any other options, we have title and count as a `ReactElement ` and index. No more options